.buildkite/scripts/validate-tag.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -euo pipefail
+
+source .buildkite/scripts/tooling.sh
+
+transformTagAndValidate() {
+    version=$1
```

Comment:
nit. use local variables inside the function
```suggestion
    local version=$1
```